oie/ja-jp/content/topics/dashboard/dashboard.htm

```diff
@@ -124,7 +124,7 @@
                                 </div>
                             </div>
                             <div class="title-bar-layout outer-row">
-                                <div class="logo-wrapper"><a class="logo" href="https://support.okta.com/help/s/?language=ja" alt="Okta"></a>
+                                <div class="logo-wrapper"><a class="logo" href="https://help.okta.com/okta_help.htm?id=csh-index" alt="Okta"></a>
```

Comment:
If I navigate directly to https://help.okta.com/okta_help.htm?id=csh-index I always get the content in Engish, even if I had selected Japanese in the language switcher earlier. Do we need some url param to indicate the language?